webapp/fmprod/src/main/webapp/WEB-INF/web.xml

```diff
@@ -72,7 +72,7 @@
     <init-param>
       <param-name>jaxrs.serviceClasses</param-name>
       <param-value>
-        org.apache.oodt.cas.product.jaxrs.services.CasProductJaxrsService
```

Comment:
Just to clarify. This means we will not be using `CasProductJaxrsService` in the future and it will be replaced by `FileManagerJaxrsServiceV2`?

Reply:
Yes Indeed. But at the current situation to maintain back-compatibility I separate v2 APIs completely from existing ones. [PR#97](https://github.com/apache/oodt/pull/97/commits/d88b9500d76fa98979ce5213e904f5fa2e03c0fc) fixes this.